aspnetcore/blazor/file-uploads.md

```diff
@@ -336,7 +379,7 @@ The following `FileUpload2` component:
 
 :::moniker range=">= aspnetcore-8.0"
 
-If the component limits file uploads to a single file at a time or if the component only adopts interactive client-side rendering (CSR, `InteractiveWebAssembly`), the component can avoid the use of the `LazyBrowserFileStream` and use a <xref:System.IO.Stream>. The following demonstrates the changes for the `FileUpload2` component:
+If the component limits file uploads to a single file at a time or if the component only adopts client-side rendering (CSR, `InteractiveWebAssembly`), the component can avoid the use of the `LazyBrowserFileStream` and use a <xref:System.IO.Stream>. The following demonstrates the changes for the `FileUpload2` component:
```

Comment:
@guardrex It's not really clear to me what problem `LazyBrowserFileStream` is trying to solve. Why is it needed?

Reply:
@lewing @javiercn It looks like `LazyBrowserFileStream` was introduced per @MackinnonBuck's [recommendation](https://github.com/dotnet/AspNetCore.Docs/pull/33357#pullrequestreview-2240989426) to work around this framework bug: https://github.com/dotnet/aspnetcore/issues/47301. Perhaps we should prioritize getting https://github.com/dotnet/aspnetcore/issues/47301 fixed for .NET 10 so that this workaround is no longer needed?

Reply:
I'll make a 10.0 tracking entry for https://github.com/dotnet/aspnetcore/issues/47301 to keep it on my radar.